From bb5f6e433145838a217a1e245e31bb207c504c2b Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Tue, 15 Jun 2021 10:38:27 +0300
Subject: [PATCH] [SW-44018] gaudi: add missing call to e2e credit

Last patch which added e2e credit routine in master,
had missed the actual call to the routine during
init. This patch fix it.

Change-Id: I4795ff3e71522bea3f4a8bf9f2a63bf8316332fc
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/preboot/preboot.c b/preboot/preboot.c
index b7d6436..1274637 100644
--- a/preboot/preboot.c
+++ b/preboot/preboot.c
@@ -395,6 +395,7 @@
 	/* Get the VendorID before we lock the access to configuration space */
 	preboot_retrieve_device_id_from_pcie();
 
+	preboot_e2e_credits_init();
 	preboot_hbm_credits_init();
 	preboot_rate_limiter_init();
 
